src/main/java/hlf/java/rest/client/service/impl/ChaincodeOperationsServiceImpl.java

```diff
@@ -94,26 +96,29 @@ public String getCurrentSequence(
           queryChaincodeDefinitionProposalResponses =
               channel.lifecycleQueryChaincodeDefinition(
                   queryLifecycleQueryChaincodeDefinitionRequest, peers);
-      Set<Long> sequenceNumbers = new HashSet<>();
+      long sequence = -1L;
       for (LifecycleQueryChaincodeDefinitionProposalResponse response :
           queryChaincodeDefinitionProposalResponses) {
-        if (response.getVersion().equals(chaincodeVersion)) {
-          sequenceNumbers.add(response.getSequence());
+        if (response.getStatus() == ProposalResponse.Status.SUCCESS
```

Comment:
[nitpick] always prefer equals() for checking value equality. This if block does equality check in two different ways.

Reply:
change done